src/main/java/com/kryptnostic/rhizome/hazelcast/objects/SetProxy.java

```diff
@@ -4,7 +4,7 @@
 
 public interface SetProxy<K, T> extends Set<T> {
 
-    public static final String KEY_COLUMN_NAME   = "setId";
+    public static final String KEY_COLUMN_NAME   = "setid";
     public static final String VALUE_COLUMN_NAME = "data";
```

Comment:
What's the motivation for changing this (though I guess it's not a big deal because we're planning to delete the entire prod database right before migrating)